src/components/ReviewSingleWithComments.jsx

```diff
@@ -12,12 +13,18 @@ const ReviewSingleWithComments = () => {
       setReview(review);
       setIsLoading(false);
     });
-  }, []);
+  }, [review_id]);
   if (isLoading) return <p>Loading review...</p>;
```

Comment:
I'm not such a fan of this - ternaries are much more commonplace in real world React development. I would use a ternary, but again, up to you.